### PR TITLE
Update instructions for current site structure

### PR DESCRIPTION
**Pull request by mfurtak**
screen shot 2016-08-31 at 10 53 21 am

screen shot 2016-08-31 at 10 53 06 am

### CODE REVIEW

File: supply/README.md [hunk 1]
```diff
@@ -78,7 +78,9 @@ Setup consists of setting up your Google Developers Service Account
 - Open the [Google Play Console](https://play.google.com/apps/publish/)
 - Select **Settings** tab, followed by the **API access** tab
 - Click the **Create Service Account** button and follow the **Google Developers Console** link in the dialog
-- Click **Create credentials** and select **Service account**
+- Click the **Create Service account** button at the top of the developers console screen
+- Provide a name for the service account
+- Check the **Furnish a new private key** checkbox
```
Comment (KrauseFx) [inline supply/README.md]: Furnish?

Comment (KrauseFx) [inline supply/README.md]: Seems right, but sounds funny

**Comment by KrauseFx** [comment]
Looks good

File: supply/README.md (hunk 1 above)
Comment (mfurtak) [inline supply/README.md]: Haha, yeah, it's Google's language :wink:

**Pull request merged**